src/stories/static_data/tileData.js

```diff
@@ -0,0 +1,102 @@
+export const tileData = [
+  {
+    "url": "/services/hazardous-waste-dropoff",
+    "text": "Drop off household hazardous waste and other recyclables"
+  },
+  {
+    "url": "/services/compost-pickup",
+    "text": "Get ready for curbside compost pickup"
+  },
+  {
+    "url": "/services/pickup-free-paint",
+    "text": "Pick up free paint and other household items"
+  },
+  {
+    "url": "/services/bulk-item-pickup",
+    "text": "Get ready for curbside bulk item pickup"
+  },
+  {
+    "url": "/services/trash-recycling-pickup",
+    "text": "Look up your trash, recycling, and compost pickup days"
+  }
+]
+
+export const tileGroupData = {
+  "edges": [
+    {
+      "node": {
```

Comment:
Mentioned this elsewhere, but I think storybook yet again shows us how we've tightly coupled our components -- this time to graphQL style data.